mobile/src/navigations/onboarding-navigation/onboarding-navigation.tsx

```diff
@@ -17,15 +26,48 @@ const OnboardingNavigation: React.FC = () => {
                 header: ({ navigation }): React.ReactNode => (
                     <Header navigation={navigation} />
                 ),
+                drawerStyle: {
+                    width: 330,
+                },
             }}
+            drawerContent={(
+                props: DrawerContentComponentProps,
+            ): React.ReactNode => <Steps {...props} />}
         >
             <Drawer.Screen
                 name={OnboardingScreenName.PROFILE}
                 component={Profile}
+                initialParams={{
+                    stepState: 'focused',
+                }}
             />
             <Drawer.Screen
                 name={OnboardingScreenName.BSA_BADGES}
                 component={BsaBadges}
+                initialParams={{
+                    stepState: 'focused',
```

Comment:
Did you mean `stepState: OnboardingStepState.FOCUSED`?
Icon and text styles don't work as they should, because I don't pass isFocused from Steps component but hardcoding it. But as I understood we will do it conditionally later. So it's not a big problem for now, I suppose.